clients/csharp-uwp/UWPVoiceAssistantSample/KeywordRegistration.cs

```diff
@@ -153,11 +153,6 @@ public bool KeywordEnabledByApp
         /// <returns>A <see cref="Task"/> that returns on successful keyword setup.</returns>
         public async Task<ActivationSignalDetectionConfiguration> GetOrCreateKeywordConfigurationAsync()
         {
-            if (string.IsNullOrWhiteSpace(this.KeywordActivationModelFilePath) && string.IsNullOrWhiteSpace(this.ConfirmationKeywordModelPath))
-            {
-                await LocalSettingsHelper.CopyConfigAndAssignValues();
-            }
-
```

Comment:
This was added because this operation may not have completed for the first time by this point.

App startup triggers a fire-and-forget that copies and assigns these values then triggers this method as a fire and forget when it creates the keyword registration. Is there a change that fixes this a different way?

Reply:
Entry points do an (inelegant but effective) Task.Run(async () => await InitAsync()).Wait() now.